generators/pythonv2/codegen/src/ast/__test__/__snapshots__/PythonFile.test.ts.snap

```diff
@@ -0,0 +1,39 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`PythonFile > Add a Method 1`] = `
+"def test_method() -> str:
+    pass
+
+"
+`;
+
+exports[`PythonFile > Add a class with a reference that uses a python standard library reference 1`] = `
+"from itertools import chain
+
+class TestClass:
+    flat_list = list(itertools.chain([[1, 2], [3, 4]]))
+"
+`;
+
+exports[`PythonFile > Add a class with a reference that uses a relative import 1`] = `
+"from ..my_module.level_1 import OtherClass
+from ..my_module.level_1.level_2 import DeepClass
+
+class TestClass(OtherClass):
+    pass
+
+class DeeplyNestedTestClass(DeepClass):
+    pass
+"
+`;
```

Comment:
This is probably the most interesting use-case here. It shows the upwards traversal of relative imports to the module root.